Arduino/Svc/LifeLed/LifeLed.cpp

```diff
@@ -0,0 +1,48 @@
+// ======================================================================
+// \title  LifeLed.cpp
+// \author mstarch
+// \brief  cpp file for LifeLed component implementation class
+// ======================================================================
+
+#include "Arduino/Svc/LifeLed/LifeLed.hpp"
+#include "FpConfig.hpp"
+
+namespace Arduino {
+
+// ----------------------------------------------------------------------
+// Component construction and destruction
+// ----------------------------------------------------------------------
+
+LifeLed ::LifeLed(const char* const compName) : LifeLedComponentBase(compName), m_blink_count(0), m_pin(Arduino::DEF_LED_BUILTIN), m_configured(false) {}
```

Comment:
For boards with no onboard LED and with definition to the `LED_BUILTIN` macro, `Arduino::DEF_LED_BUILTIN` is undefined.

`Arduino::DEF_LED_BUILTIN` is defined in `FprimeArduino.hpp` iff `LED_BUILTIN` is defined:
```hpp
enum ARDUINO_DEFINITION_REMAP {
    DEF_HIGH = HIGH,
    DEF_LOW = LOW,
    DEF_INPUT = INPUT,
    DEF_OUTPUT = OUTPUT,
    DEF_INPUT_PULLUP = INPUT_PULLUP,
#ifdef LED_BUILTIN
    DEF_LED_BUILTIN = LED_BUILTIN
#endif //LED_BUILTIN
};
```